src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+import Book from '../../components/Book';
+
+// actions
+
+const ADD = 'ADD';
```

Comment:
- [ ] I kindly suggest that the value of each action type should follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Please make changes to your remove book action type also

## EXAMPLE
```
const ADD_BOOK = bookstore/books/ADD_BOOK

const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

Reply:
Thanks for the suggestion, done!